Resources/Prototypes/Entities/Stations/base.yml

```diff
@@ -17,6 +17,7 @@
     - type: StationBankAccount
     - type: StationCargoOrderDatabase
     - type: StationCargoBountyDatabase
+    - type: StationLogisticStats # DeltaV - Tracks statistics related to mail and income
```

Comment:
true.

Reply:
actually i think i remembereed the reason i put it on Cargo was because "somewhere down the line" i imagined adding actual cargo income related stats there but that is both nebulous and unnecessary to prepare for "somewhere down the line" lets see if i can move it and itll work without errors